include/logger/loggerConfig.h

```diff
@@ -357,17 +358,19 @@ typedef struct _GPSConfig {
 #define DEFAULT_GPS_SATELLITE_CONFIG {"GPSSats", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
 #define DEFAULT_GPS_QUALITY_CONFIG {"GPSQual", "", 0, 5, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
 #define DEFAULT_GPS_DOP_CONFIG {"GPSDOP", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 1, 0}
-
-#define DEFAULT_GPS_CONFIG {                   \
-		DEFAULT_GPS_LATITUDE_CONFIG,           \
-		DEFAULT_GPS_LONGITUDE_CONFIG,          \
-		DEFAULT_GPS_SPEED_CONFIG,              \
-		DEFAULT_GPS_DISTANCE_CONFIG,           \
-		DEFAULT_GPS_ALTITUDE_CONFIG,           \
-		DEFAULT_GPS_SATELLITE_CONFIG,          \
-		DEFAULT_GPS_QUALITY_CONFIG,            \
-		DEFAULT_GPS_DOP_CONFIG                 \
-         }
+#define DEFAULT_GPS_HEADING_CONFIG {"GPSHeading", "Degrees", 0, 360, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
```

Comment:
Compass channel is called "Compass"

Brent Picasso
Autosport Labs
Technology for Race and Street
On Jul 30, 2015 7:45 AM, "Andrew Stiegmann" <notifications@github.com>
wrote:

> In include/logger/loggerConfig.h
> <https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/70#discussion_r35877981>
> :
>
> > @@ -357,17 +358,19 @@ typedef struct _GPSConfig {
> >  #define DEFAULT_GPS_SATELLITE_CONFIG {"GPSSats", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
> >  #define DEFAULT_GPS_QUALITY_CONFIG {"GPSQual", "", 0, 5, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
> >  #define DEFAULT_GPS_DOP_CONFIG {"GPSDOP", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 1, 0}
> > -
> > -#define DEFAULT_GPS_CONFIG {                   \
> > -		DEFAULT_GPS_LATITUDE_CONFIG,           \
> > -		DEFAULT_GPS_LONGITUDE_CONFIG,          \
> > -		DEFAULT_GPS_SPEED_CONFIG,              \
> > -		DEFAULT_GPS_DISTANCE_CONFIG,           \
> > -		DEFAULT_GPS_ALTITUDE_CONFIG,           \
> > -		DEFAULT_GPS_SATELLITE_CONFIG,          \
> > -		DEFAULT_GPS_QUALITY_CONFIG,            \
> > -		DEFAULT_GPS_DOP_CONFIG                 \
> > -         }
> > +#define DEFAULT_GPS_HEADING_CONFIG {"GPSHeading", "Degrees", 0, 360, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
>
> How do you intend to differentiate between compass heading and GPS heading?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/70/files#r35877981>
> .
>